dhis-2/dhis-services/dhis-service-core/src/test/java/org/hisp/dhis/dataapproval/DataApprovalServiceTest.java

```diff
@@ -1778,8 +1763,8 @@ public void testWorkflows()
         assertEquals( "APPROVED_HERE level=level3 approve=F unapprove=T accept=F unaccept=F read=T", statusAndPermissions( workflow3, periodA, organisationUnitC, defaultOptionCombo ) );
     }
 
+    @Ignore
```

Comment:
Do you remember why this is ignored, is there a jira issue for it?

Reply:
Yes, because like AnalyticsServiceTest it gives unpredictable results. Thread.sleep didn't help this time. Will need to look into it together with finding solution for handling hibernate flushes. 
https://jira.dhis2.org/browse/DHIS2-5340